src/status_im/chat/handlers.cljs

```diff
@@ -291,18 +289,18 @@
 (register-handler :add-chat-loaded-callback
   (fn [db [_ chat-id callback]]
     (log/debug "Add chat loaded callback: " chat-id callback)
-    (update-in db [::chat-loaded-callbacks chat-id] conj callback)))
+    (update-in db [:chat-loaded-callbacks chat-id] conj callback)))
```

Comment:
It would be nice to have a convention for keywords. Not sure if there is one yet. I've seen a mix of namespaced and not namespaced.